tests/ProjectsTest.php

```diff
@@ -475,6 +475,9 @@ public function testCreateProjectWithBackend(string $backend): void
         ]);
 
         $this->assertEquals($backend, $project['defaultBackend']);
+        if ($project['backends'] !== []) {
```

Comment:
Ten if mi prijde zbytecny, kdyz jde o test CreateProjectWithBackend. Naopak bych testoval, ze to tam byt musi a overil i hodnotu v owner.

Reply:
Právě se mi to s defaultním nastavením pro tento test založí backend jen pro Snowflake. Ale teď mě napadá, jestli to náhodou nesouvisí s tím, co mám založené lokálně. Jdu to vyzkoušet a případně upravit.

Reply:
upraveno